bin/dev/legalscan.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
+
+for PACKAGE in $(osc ls Cloud:Platform:buildpacks:dependencies); do
```

Comment:
Are `Cloud:Platform:sources:scf`, `Cloud:Platform:sources:buildpacks` and `Cloud:Platform:sources:sidecars` no longer reviewed?

Reply:
I have wrote a simple script to accept them as soon as legaldb is done. This prevents tarballs from being recomitted even though they were not changed.